src/app/qgsmaptoolscalefeature.cpp

```diff
@@ -0,0 +1,479 @@
+﻿/***************************************************************************
+    qgsmaptoolscalefeature.cpp  -  map tool for scaling features by mouse drag
+    ---------------------
+    begin                :
```

Comment:
populate

Reply:
done, didn't put my email though, still not at ease with that...